### ISSUE TITLE
Translation of meta page description of help/faq page

### ISSUE THREAD

**Issue by equivalentideas** [OPORA/publicwhip#160]
The English description of the [Help/FAQ page](https://rada4you.org/help/faq) is:

> Parliament's confusing jargon explained in plain English.

It's defined in the code here: https://github.com/OPORA/publicwhip/blob/develop/app/views/help/faq.uk.html.haml#L2

Is this text appropriate for the new Ukrainian text on that page?

Could we please have Ukrainian text for this @orybiy @mmitre ?

**Comment by orybiy** [comment]
Parliament's confusing jargon explained in plain English.

Парламентський специфічний "жаргон", пояснений простими словами

**Comment by beastie87** [comment]
Done
